Doc/library/stdtypes.rst

```diff
@@ -4303,7 +4304,8 @@ is usually unwise to use them as dictionary keys.)
 
 Dictionaries can be created by placing a comma-separated list of ``key: value``
 pairs within braces, for example: ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098:
-'jack', 4127: 'sjoerd'}``, or by the :class:`dict` constructor.
+'jack', 4127: 'sjoerd'}``, by a dict comprehension, for example:
+``{x: x**2 for x in iterable}`` or by the :class:`dict` constructor.
```

Comment:
Split into three sentences,and add an example for the dict constructor:  ``dict(jack=4098, sjoerd= 4127}``